src/Microsoft.DocAsCode.Dfm/Rules/DfmFencesInlineRule.cs

```diff
@@ -34,6 +38,10 @@ public override IMarkdownToken TryMatch(IMarkdownParser parser, IMarkdownParsing
             var title = match.Groups["title"]?.Value;
             var pathQueryOption = ParsePathQueryString(match.Groups["option"]?.Value, match.Groups["optionValue"]?.Value);
 
+            if (!parser.Context.GetIsInTable())
+            {
+                Logger.LogWarning("Invalid inline code.", line: sourceInfo.LineNumber.ToString(), code: WarningCodes.Markdown.InvalidInlineCodeSnippet);
```

Comment:
more descriptive warning message, e.g. "Inline code snippet is only allowed inside tables"